### DISCUSSION TITLE
想请教一下 xray 如果给两台机器做中转？

### DISCUSSION THREAD

**Discussion by ssydb** [XTLS/Xray-core#456] [Q&A]
手上有两台机器 一台做落地 一台做中转 两台都已经部署了 xray 协议（用的是荒岛的教程 ”Xray+VLESS+XTLS+NginxSNI分流/443端口复用“）

网上找了一些资料都是针对 v2ray 的 自己修改了一下都不成功 所以想来问一下~

另外还有个问题 就是想用 dns 解锁奈飞服务 有没有好的方法 现在能用的方法是在客户端的情况下可以正常解锁观看 一担用路由器FQ就会出现解锁失败的情况

**Comment by ssydb** [comment]
{
    "log": {
        "loglevel": "warning"
    },
    "inbounds": [
        {
            "listen": "127.0.0.1", # 仅监听在本地防止探测到下面的50001端口
            "port": 12345, # 这里的端口对应nginx内的upstream端口
            "protocol": "vless",
            "settings": {
                "clients": [
                    {
                        "id": "uuid", # 填写你的UUID
                        "flow": "xtls-rprx-direct",
                        "level": 0
                    }
                ],
                "decryption": "none",
                "fallbacks": [
                    {
                        "dest": "23333" # 回落站点的端口号
                    }
                 ]
            },
            "streamSettings": {
                "network": "tcp",
                "security": "xtls",
                "xtlsSettings": {
                    "alpn": [
                        "http/1.1"
                    ],
                    "certificates": [
                        {
                            "certificateFile": "/usr/local/etc/xray/fullchain.pem", # 你的域名证书
                            "keyFile": "/usr/local/etc/xray/privkey.pem" # 你的证书私钥
                        }
                    ]
                }
            }
        }
    ],
    "outbounds": [
        {
            "protocol": "freedom"
        }
    ]
}

**Comment by ssydb** [comment]
上面照教程部署的 config.json

**Comment by AkinoKaede** [comment]
中转服务器出站 XTLS 即可
